kotlin/configure-subnetwork-trace/src/main/res/layout/activity_main.xml

```diff
@@ -1,202 +1,210 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Fixed margins alignments and balanced spinners. There were a couple other layout style issues which are fixed. The UI elements should now feel in place rather than floating around oddly.